numpyro/compat/pyro.py

```diff
@@ -1 +1,10 @@
+import warnings
+
+from numpyro.compat.util import UnsupportedAPIWarning
 from numpyro.primitives import module, param, plate, sample  # noqa: F401
+
+
+def get_param_store():
+    warnings.warn('NumPyro does not have a parameter store.', category=UnsupportedAPIWarning)
```

Comment:
nit: I think it will be helpful to add: "If you are doing SVI, values of `param` primitives can be obtained by using `svi.get_params()` method."